content/news/2024-02-03-bevy-0.13/index.md

```diff
@@ -872,6 +872,50 @@ let layout = render_device.create_bind_group_layout(
 );
 ```
 
+## Unload rendering assets from RAM
+
+<div class="release-feature-authors">authors: @JMS55, @mockersf, @brianreavis</div>
+
+Meshes and the textures used to define their materials take up a ton of memory:
+in many games, memory usage is the biggest limitation on the resolution and polygon count of the game!
+Moreover, actually transferring that data from the CPU (and RAM) to the GPU (and VRAM) can be a real performance bottleneck,
+especially if you're doing it from scratch every frame.
```

Comment:
I feel like people only do this in niche cases, so it’s not worth mentioning as it’s not something that is generally done.